test/integration/tags_test.exs

```diff
@@ -305,4 +305,28 @@ defmodule Solid.Integration.TagsTest do
              """
     end
   end
+
+  describe "raw" do
```

Comment:
Thanks for the tests 🏆 
Would you mind adding an integration test case here https://github.com/edgurgel/solid/tree/master/test/cases ?

Maybe something like:

```elixir
{% raw %}{{ 5 | plus: 6 }}{% endraw %} equals 11.
{% raw %} {{ {% {%  {% endraw %} equals 11.
{% raw %}{% increment counter %}{{ counter }}{% endraw %}
{% increment counter %} {{ counter }}
```

This is so that we can compare our implementation with the original implementation in Ruby

Reply:
No problem!